src/units_for_slot_test.rs

```diff
@@ -257,9 +730,21 @@ mod units_for_slot_tests {
         let rules = get_mock_rules(&categories);
         let channel = mock_channel(&rules);
 
+        let non_matching_categories: [&str; 3] = ["IAB2", "IAB9-WS1", "IAB19"];
+        let non_matching_rules = get_mock_rules(&non_matching_categories);
+        let mut non_matching_channel = mock_channel(&non_matching_rules);
+        non_matching_channel.id =
+            ChannelId::from_hex("061d5e2a67d0a9a10f1c732bca12a676d83f79663a396f7d87b3e30b9b411089")
```

Comment:
Maybe we should `impl FromStr` for `ChannelId`, this way we can remove dependencies like `hex`

Reply:
@elpiel I agree, it would be more convenient